src/lightning/pytorch/profilers/simple.py

```diff
@@ -80,15 +80,24 @@ def stop(self, action_name: str) -> None:
     def _make_report_extended(self) -> Tuple[_TABLE_DATA_EXTENDED, float, float]:
         total_duration = time.monotonic() - self.start_time
         report = [
-            (a, np.mean(d), len(d), np.sum(d), 100.0 * np.sum(d) / total_duration)
+            (
+                a,
+                torch.mean(torch.tensor(d)).item(),
+                len(d),
+                torch.sum(torch.tensor(d)).item(),
+                100.0 * torch.sum(torch.tensor(d)).item() / total_duration,
+            )
             for a, d in self.recorded_durations.items()
```

Comment:
@ishandutta0098 what is the type of the variables a and d here?

Reply:
`a` is `str` and `d` is `list`